src/dash/DashAdapter.js

```diff
@@ -309,7 +309,7 @@ function DashAdapter() {
         voAdaptations = {};
     }
 
-    function getStreamsInfo(externalManifest) {
+    function getStreamsInfo(externalManifest, limit) {
```

Comment:
@Gontran-Molotov , it could be interesting to rename limit parameter by, for instance, maxStreamsInfo

Reply:
That's indeed a more explicit variable name. Thanks for the suggestion.